tests/test_data_deletion/test_dmf_deletion.py

```diff
@@ -0,0 +1,45 @@
+import os
+from datetime import datetime
+from unittest.mock import patch, Mock, PropertyMock
+from data_deletion import ProcessedSample
+from data_deletion.DMF_data import DMFDtatDeleter
+from data_deletion.delivered_data import DeliveredDataDeleter
+from egcg_core.exceptions import ArchivingError
+from tests import TestProjectManagement
+from tests.test_data_deletion import TestDeleter, patched_patch_entry
+
+ppath = 'data_deletion.'
+
+
+class TestDMFDtatDeleter(TestDeleter):
+    file_exts = (
+        'bam', 'bam.bai', 'vcf.gz', 'vcf.gz.tbi', 'g.vcf.gz', 'g.vcf.gz.tbi', 'R1.fastq.gz',
+        'R2.fastq.gz', 'R1_fastqc.html', 'R2_fastqc.html'
+    )
+    samples = (
+        Mock(sample_id='this', files_to_purge=['folder_this'], files_to_remove_from_lustre=['a_file'], size_of_files=2),
+        Mock(sample_id='that', files_to_purge=['folder_that'], files_to_remove_from_lustre=['another_file'], size_of_files=4)
+    )
```

Comment:
Are these class attributes used for anything?

Reply:
nop they aren't and have been removed